frontend/components/watch-card.js

```diff
@@ -116,6 +150,9 @@ const WatchCard = observer((props) => {
         .target-row:hover {
           background-color: hsl(0, 0%, 98%); // white-bis
         }
+        .double-dots {
+          margin-right: 0.75rem;
```

Comment:
Thôi cứ để 1rem đi, cho tròn
```suggestion
          margin-right: 1rem;
```